frontend/public/components/pod-exec.jsx

```diff
@@ -194,16 +195,21 @@ const PodExec_ = connectToFlags(FLAGS.OPENSHIFT)(
 
       return (
         <div>
+          {infoMessage && infoMessage}
           <div className="co-toolbar">
             <div className="co-toolbar__group co-toolbar__group--left">
               <div className="co-toolbar__item">{t('public~Connecting to')}</div>
               <div className="co-toolbar__item">
-                <Dropdown
-                  className="co-toolbar__item-dropdown"
-                  items={_.mapValues(containers, nameWithIcon)}
-                  title={nameWithIcon(activeContainer || <LoadingInline />)}
-                  onChange={this.onChangeContainer}
-                />
+                {containers.length > 1 ? (
+                  <Dropdown
+                    className="co-toolbar__item-dropdown"
+                    items={_.mapValues(containers, nameWithIcon)}
+                    title={nameWithIcon(activeContainer || <LoadingInline />)}
+                    onChange={this.onChangeContainer}
+                  />
+                ) : (
+                  nameWithIcon(containers?.[0])
```

Comment:
No need for `?` here since you're not using it in `containers.length`, which would blow up if undefined.